python/cuml/test/test_metrics.py

```diff
@@ -1080,3 +1084,46 @@ def test_pairwise_distances_output_types(input_type, output_type, use_global):
             assert isinstance(S, np.ndarray)
         elif output_type == "cupy":
             assert isinstance(S, cp.core.core.ndarray)
+
+
+@pytest.mark.parametrize("input_type, nrows, ncols",
+                         [
+                             unit_param(30, 10, 7),
+                             quality_param(5000, 100, 50),
+                             stress_param(500000, 200, 100)
+                         ])
+@pytest.mark.parametrize("input_type", ["cudf", "cupy"])
+def test_hinge_loss(input_type, nrows, ncols, n_info):
```

Comment:
```suggestion
@pytest.mark.parametrize("nrows, ncols, n_info",
                         [
                             unit_param(30, 10, 7),
                             quality_param(5000, 100, 50),
                             stress_param(500000, 200, 100)
                         ])
@pytest.mark.parametrize("input_type", ["cudf", "cupy"])
def test_hinge_loss(nrows, ncols, n_info, input_type):
```

This will fix your current test error with the message `E   ValueError: duplicate 'input_type'`